build.sbt

```diff
@@ -442,6 +443,83 @@ lazy val integration = project
     testkit
   )
 
+lazy val websiteSettings = Seq(
+  micrositeName := "scalafix",
+  micrositeDescription := "A rewrite and linting tool for Scala",
+  micrositeBaseUrl := "scalafix",
+  micrositeDocumentationUrl := "docs",
+  micrositeHighlightTheme := "atom-one-light",
+  micrositeHomepage := "https://scalacenter.github.io/scalafix/",
+  micrositeOrganizationHomepage := "https://scala.epfl.ch/",
+  micrositeTwitterCreator := "@scala_lang",
+  micrositeGithubOwner := "scalacenter",
+  micrositeGithubRepo := "scalafix",
+  micrositeGithubToken := sys.env.get("GITHUB_TOKEN"),
+  micrositePushSiteWith := GitHub4s,
+  ghpagesNoJekyll := false,
+  micrositeGitterChannel := true,
+  micrositeFooterText := None,
+  micrositeFooterText := Some(
+    """
+      |<p>© 2017 <a href="https://github.com/scalacenter/scalafix#team">The Scalafix Maintainers</a></p>
+      |<p style="font-size: 80%; margin-top: 10px">Website built with <a href="https://47deg.github.io/sbt-microsites/">sbt-microsites © 2016 47 Degrees</a></p>
+      |""".stripMargin
+  ),
+  micrositePalette := Map(
+    "brand-primary"     -> "#0D2B35",
+    "brand-secondary"   -> "#203F4A",
+    "brand-tertiary"    -> "#0D2B35",
+    "gray-dark"         -> "#453E46",
+    "gray"              -> "#837F84",
+    "gray-light"        -> "#E3E2E3",
+    "gray-lighter"      -> "#F4F3F4",
+    "white-color"       -> "#FFFFFF"
+  ),
+  micrositeConfigYaml := ConfigYml(
+    yamlCustomProperties = Map(
+      "githubOwner" -> micrositeGithubOwner.value,
+      "githubRepo" -> micrositeGithubRepo.value,
+      "docsUrl" -> micrositeDocumentationUrl.value,
+      "callToActionText" -> "View Documentation",
+      "callToActionUrl" -> micrositeDocumentationUrl.value,
+      "scala212" -> scala212,
+      "scala211" -> scala211,
+      "stableVersion" -> (stableVersion in coreJVM).value, // TODO(gabro): there must be a simpler way...
+      "scalametaVersion" -> scalametaV,
+      "supportedScalaVersions" -> Seq(scala211, scala212), // TODO(gabro): how to reference the one defined in buildInfoKeys?
```

Comment:
We can create a `val supportedScalaVersions = List(...)` that we reuse. Right here we can do `mkString(", ")` to avoid the `| join ","` in the jekyll template.